### DISCUSSION TITLE
Unable to set allow_insecure	to true

### DISCUSSION THREAD

**Discussion by javiercj93** [databendlabs/databend#15539] [Q&A] [answered]
Not sure if I am following your documentation correctly but based on this [page](https://www.databend.com/blog/2023-10-18-databend-configuration-guide/) in order to configure some of the options in a query node we have to do something like this:

"configuration domain" + "configuration item."

When spinning up the docker container for a local test I want to do with minio and databend I am setting this env variable:

-e QUERY_STORAGE_ALLOW_INSECURE=true \

However I keep getting error: APIError: ResponseError with 3903: copy from insecure storage is not allowed

Any idea why it is not working? Thanks in advance

**Comment by sundy-li** [comment, marked as answer]
```
./databend-query -c .vscode/query.toml  --storage-allow-insecure
```
